common/scripted_triggers/wc_scripted_plot_triggers.txt

```diff
@@ -0,0 +1,80 @@
+can_be_killed_by_poisoned_wine_trigger = {
+	can_be_drunkard_trigger = yes
+}
+can_be_killed_by_falling_trigger = {
+	is_humanoid_trigger = yes
+	dark_public_religion_trigger = no
+	
+	can_fly_trigger = no
+}
+can_be_killed_by_archer_ambush_trigger = {
+	#Tribal/nomadic ruler touring in a carriage... sounds weird... don't know why Paradox haven't disabled it
+	is_tribal = no
+	is_nomadic  no
+	
+	is_humanoid_trigger = yes
+	dark_public_religion_trigger = no
+}
+can_be_killed_by_manure_explosion_trigger = {
+	#Tribal/nomadic ruler touring and entering taverns... sounds weird... don't know why Paradox haven't disabled it
+	#Let's assume tribal monarchs have taverns and sometimes enter them
+	NOT = { government = tribal_government }
+	is_nomadic  no
+	
+	is_humanoid_trigger = yes
+	dark_public_religion_trigger = no
+}
+can_be_killed_by_infanticide_trigger = {
+	# Everyone? It's the only way to kill a character under 5 years old
+}
+can_be_killed_by_prisoner_break_trigger = {
+	# Everyone? It's the only way to kill a prisoner
+}
+can_be_killed_by_bathtub_drowning_trigger = {
+	is_humanoid_trigger = yes
+	can_breath_underwater_trigger = no
+}
+can_be_killed_by_boating_accident_trigger = {
```

Comment:
> Whispers have reached my ears that [Root.PlotTarget.GetTitledName] is due to embark on a little boat trip. I have it on good authority that [Root.PlotTarget.GetSheHe], like many children, has not yet learned to swim. If the boat was to flounder... well, these things happen all the time, don't they?

Can't they just fly away?

Reply:
Make sense. Added harpies limit in eef083a90c49655de63b680162b54e0209b47683.